changelogs/fragments/1109-organizations-role.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - new ``organizations`` role to manage organizations (https://github.com/theforeman/foreman-ansible-modules/issues/1109)
```

Comment:
Is double `` intentional?

Reply:
yes, it ends up in reStructuredText which uses double for code…